examples/decoding/plot_decoding_rsa.py

```diff
@@ -0,0 +1,179 @@
+"""Representational Similarity Analysis is used to perform summary statistics
+on supervised classifications where the number of classes is relatively high.
+It consists in characterizing the structure of the confusion matrix to infer
+the similarity between brain responses and serve as a proxy for characterizing
+the space of mental representations [1-3].
+
+In this example, we perform RSA on spiking data from [4] where the macaque was
+presented to seven classes of pictures located in one of three spatial
+positions.
+
+References
+----------
+[1] Shepard, R. "Multidimensional scaling, tree-fitting, and clustering."
+ Science 210.4468 (1980): 390-398.
+[2] Laakso, A. & Cottrell, G.. "Content and cluster analysis:
+ assessing representational similarity in neural systems." Philosophical
+ psychology 13.1 (2000): 47-76.
+[3] Kriegeskorte, N., Marieke, M., & Bandettini.  P. "Representational
+ similarity analysis-connecting the branches of systems neuroscience."
+ Frontiers in systems neuroscience 2 (2008): 4.
+[4] Zhang, Ying, et al. "Object decoding with attention in inferior temporal
+ cortex." Proceedings of the National Academy of Sciences 108.21 (2011):
+ 8850-8855.
```

Comment:
I find that this paper introduces the topic of RSA very well:

Kriegeskorte, N., Mur, M., & Bandettini, P. a. (2008). Representational similarity analysis - connecting the branches of systems neuroscience. Frontiers in Systems Neuroscience, 2(November), 4. http://doi.org/10.3389/neuro.06.004.2008